mne/viz/evoked.py

```diff
@@ -1339,20 +1339,36 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
     return fig
 
 
-def _ci(arr, ci):
+def _ci(arr, ci, method="parametric"):
     """Calculate the `ci`% parametric confidence interval for `arr`.
 
     Aux function for plot_compare_evokeds.
     """
     from scipy import stats
-    mean, sigma = arr.mean(0), stats.sem(arr, 0)
-    # This is highly convoluted to support 17th century Scipy
-    # XXX Fix when Scipy 0.12 support is dropped!
-    # then it becomes just:
-    # return stats.t.interval(ci, loc=mean, scale=sigma, df=arr.shape[0])
-    return np.asarray([stats.t.interval(ci, arr.shape[0],
-                       loc=mean_, scale=sigma_)
-                       for mean_, sigma_ in zip(mean, sigma)]).T
+
+    if method not in ["bayesian", "parametric"]:
+        # Add warning?
+        method = "parametric"  # Return to parametric if wrong value
+
+    if method == "bayesian":
+        ci_band = []
+        for data in arr.T:
+            res_mean, res_var, res_sigma = stats.bayes_mvs(data, alpha=ci)
+            interval = res_mean[1]
+            ci_band.append(interval)
+        ci_band = np.asarray(ci_band).T
```

Comment:
Something like this is equivalent, see if you agree it's cleaner

`ci_band = np.apply_along_axis(lambda x: stats.bayes_mvs(x, alpha=ci)[0][1], -1, arr)`